h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonField.java

```diff
@@ -22,13 +22,28 @@
 
 /**
  * GeoJson fields used by the standard.
+ *
+ * Some of them will only be used by GeoJSON
+ * files of the 2008 specification.
+ * ("CRS", "CRS_URN_EPSG", "CRS_URN_OGC": These
+ * fields are no longer useful for the new
+ * standard RFC 7946 because it uses WGS 84
+ * by default.
+ * "LINK": link object doesn't exist in RFC 7946.)
+ *
+ * Although, these fields will be kept for
+ * compatibility reasons and indicated "2008" in
+ * comment.
+ *
+ * New field "BBOX" is added for interoperability
+ * reasons.
  * 
  * @author Erwan Bocher
  */
 public class GeoJsonField {
     
     static String NAME="name";
-    static String CRS ="crs";
+    static String CRS ="crs"; // 2008
```

Comment:
Every field used only in the 2008 specification of GeoJson will be commented "2008" aside. I explained this in the class header to avoid repetition in the code which could make your reading more complicated.